index.php

```diff
@@ -28,17 +28,17 @@
 /**
  *
  * Load the configuration file which contains our DB credentials.
- * Ignore errors. If the file doesn't exist or is empty, we are running in installation mode.
+ * Ignore errors. If the file doesn't exist or is empty, we are running in
+ * installation mode.
  *
  */
 
 $install = ((file_exists('.htconfig.php') && filesize('.htconfig.php')) ? false : true);
 
-@include(".htconfig.php");
-
-
-
-
+// Only load config if found, don't surpress errors
+if (!$install) {
+	include(".htconfig.php");
```

Comment:
I see it on more places than just here. Let's put this into separate PR, else this will become to big.